### DISCUSSION TITLE
Send welcome email greyed out

### DISCUSSION THREAD

**Discussion by shaunm001** [requarks/wiki#5440] [General]
Hi, trying to create a new user with the "Send a welcome email" option is greyed out:

![Capture](https://user-images.githubusercontent.com/27734187/177040443-693d1a4b-cdd2-451c-90ab-c030a1473ba7.JPG)

Tried with Local user type and Azure Active Directory user type, same problem.

When I go to my Mail settings, everything looks to be set up correctly and im able to send a test email using the Send Test Email button. 

Any thoughts on why we're not able to use the "Send a welcome email" option when creating new user accounts?

**Comment by tayeep** [comment]
The answer is here:
https://github.com/requarks/wiki/discussions/4876

**Reply by shaunm001** [reply]
Ahhh...thank you so much.

**Reply by thoni56** [reply]
Everything that is greyed out because it is not available yet should have the same "Warning" that exists in many places already "Coming soon!"

This also made me search and re-check my email settings...
